reference/gradle-plugin-configuration.md

```diff
@@ -146,6 +146,14 @@ rewrite {
 }
 ```
 
+If desired, `rewriteDryRun` can be configured so that when `check` runs, `rewriteDryRun` does too:
+
+```gradle
+tasks.named("check").configure {
+    dependsOn(tasks.named("rewriteDryRun"))
+}
```

Comment:
Also include a code snippet that shows how to make dryRun fail if it produces any output, and describe how this can be used to create a gate in CI. I can help you come up with such a snippet if you're not sure how.

Reply:
There's a snippet included right just above this with a snippet about `failOnDryRunResults = true`, if that's what's intended? https://github.com/openrewrite/rewrite-docs/pull/50/files#diff-2825925773c74bc72ef106a6ca7cf732c94644735aacadf104a4324ea0f8be73R140-R146

Can rework the snippet, or have the snippet show both being used together if that's better?